src/modules/createHTMLelement.js

```diff
@@ -0,0 +1,39 @@
+const createHTMLElement = (newElement, className, id, inHTML, parent) => {
+  const element = document.createElement(`${newElement}`);
+  element.className = `${className}`;
+  element.id = `${id}`;
+  element.innerHTML = `${inHTML}`;
+  parent.appendChild(element);
+  return element;
+};
+
+const createInputElement = (newElement, className, id, type, name, value, parent) => {
+  const element = document.createElement(`${newElement}`);
+  element.className = `${className}`;
+  element.id = `${id}`;
+  element.type = `${type}`;
+  element.name = `${name}`;
+  element.value = `${value}`;
+  parent.appendChild(element);
+  return element;
+};
```

Comment:
To avoid repeating yourself, you can declare the `element` as global once, use it in your code for proper modularity, and follow the KISS best practices.

Reply:
thanks for the suggestion I will implement the required changes to improve my code or good comment you can think off